src/client.js

```diff
@@ -476,6 +476,113 @@ class ConnectionParameters {
 
 }
 
+// ========================================================================================
+// File Uploader
+// ========================================================================================
+
+const fileUploader = (client) => {
+  /**
+   * Will increament the counter
+   * @returns {Promise<number|*>}
+   */
+  const increaseValue = async () => {
+      const xtkCounter= await client.NLWS.xtkCounter.increaseValue({name: 'xtkResource'});
+      return xtkCounter
+  }
+
+  const createFileRes = (counter, data) => {
+      return {
+          internalName: 'RES' + counter,
+          md5: data[0].md5,
```

Comment:
why data[0] ? what if the array is empty? has more than 1 element?
This needs a short comment explaining the intent and the rationale of passing an array of 1 element.